dataflow/src/main/java/org/checkerframework/dataflow/analysis/Analysis.java

```diff
@@ -29,6 +29,17 @@
         BACKWARD
     }
 
+    /**
+     * In calls to {@code Analysis#runAnalysisFor}, whether to return the store before or after the
+     * given node.
+     */
+    public static enum PreOrPost {
```

Comment:
How about either calling this `BeforeOrAfter` or renaming the enum values to `PRE` and `POST`? Mixing the names is inconsistent.